torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2720,21 +2736,24 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 device_type='cuda', dtypes=[torch.cdouble], active_if=IS_WINDOWS),
                    )),
     OpInfo('addmm',
-           dtypes=floating_types(),
+           variant_test_name='without_non_fusible_nodes',
```

Comment:
We can get away without giving this a variant_test_name, right?

Would you add a comment for why there are two addmm OpInfos?